swarm/storage/mru/resource.go

```diff
@@ -121,139 +138,93 @@ type resource struct {
 	frequency  uint64
 	version    uint32
 	data       []byte
+	publicKey  [publicKeyLength]byte
 	updated    time.Time
 }
 
 // TODO Expire content after a defined period (to force resync)
-func (self *resource) isSynced() bool {
-	return !self.updated.IsZero()
+func (r *resource) isSynced() bool {
+	return !r.updated.IsZero()
 }
 
-func (self *resource) NameHash() common.Hash {
-	return self.nameHash
+func (r *resource) NameHash() common.Hash {
```

Comment:
This need a comment

Reply:
well, `resource` is not exported

Reply:
so you dont need to have exported function name